exercises/say/test.ml

```diff
@@ -0,0 +1,201 @@
+open Core.Std
+open OUnit2
+open Say
+
+let print_result = function
+  | None -> "None"
+  | Some(x) -> x
+                 
+let tests =
+  ["cannot say numbers below 0">::(fun _ ->
+     assert_bool "-1"
+                  (in_english (-1L) |> Option.is_none)
+   );
+   "cannot say numbers above 1 trillion">::(fun _ ->
+     assert_bool "1 trillion"
+                 (in_english 1_000_000_000_000L |> Option.is_none)
+   );
+   "zero">::(fun _ ->
+     assert_equal ~printer:print_result
+                  (Some "zero")
+                  (in_english 0L)
+   );
```

Comment:
A lot of the assertions have the same structure. I.e.

```ocaml
(fun _ ->
    assert_equal ~printer:print_result (Some "zero") (in_english 0L)
)
```

Other exercises introduced a function that so that the boilerplate noise is reduced. In this case that could be something like

```ocaml
let ae exp got _test_ctxt = assert_equal ~printer:print_result exp got
```